gnat2goto/driver/goto_utils.adb

```diff
@@ -188,6 +189,28 @@ package body GOTO_Utils is
       return Value_Arg;
    end Create_Fun_Parameter;
 
+   function Compute_Memop_Size (Num_Elem : Irep; Element_Type_Size : Uint;
+                                 Index_Type : Irep;
+                                 Source_Loc : Source_Ptr := No_Location)
+                                 return Irep is
+      Member_Size : constant Irep :=
+        Make_Constant_Expr (Source_Location => Source_Loc,
+                            I_Type          => Index_Type,
+                            Range_Check     => False,
+                            Value           =>
+                       Convert_Uint_To_Hex (Value     => Element_Type_Size / 8,
+                                                   Bit_Width => 32));
```

Comment:
NB the below (parent caller of this function) is called with Esize as so:
           Make_Memcpy_Function_Call_Expr (Destination       => Data_Mem_Expr,
                                          Source            => Literal_Address,
                                           Num_Elem          => Len_Expr,
                                           Element_Type_Size =>
                                             Esize (Element_Type_Ent),
                                           Source_Loc        => Source_Loc);

Esize returns size in bits, we want bytes, therefore the division by 8. 
The Uint "element_type_size" is in bits - might be helpful to clarify for novices.

Reply:
Ok.